src/components/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as Calculator } from './Calculator/Calculator';
+export { default as Home } from './Home/Home';
```

Comment:
- **[OPTIONAL]** I think you should remove this unused file to keep your project neat